testutil.go

```diff
@@ -281,6 +276,11 @@ func NewTestLedger(t testing.TB, cfg TestLedgerConfig) *TestLedger {
 
 	kv, cleanup := store.NewTestKV(t, "level", path, kvOpts...)
 	ledger := NewLedger(kv, client, WithoutGC())
+
+	if ledger == nil {
+		panic("Error creating new test ledger")
```

Comment:
It would be better to use `t.Fatal`` in this case.

Reply:
actually this won't help either since `log.Fatal` in `NewLedger` exits and since logger in not initialized there is no message.
I guess we need to change how ledger is created and return error from `NewLedger` and deal with it on upper level.